1-exercises/B-setting-values/exercise2.js

```diff
@@ -25,7 +25,10 @@ let student = {
   - Use bracket notation to change the value of hasPassed
 */
 
-// write code here
+if (student["attendance"] = 90 && student["examScore"] > 60){
```

Comment:
The check on attendance should be 90 or more in this if condition. Be aware than `student["attendance"] = 90` doesn't compare the two but assigns 90 to student.attendance. You'd want triple equals rather than a single equals.